Colorpsace: Update OCIO config options

jakubjezek001 opened the pull request:
## Changelog Description
Added new ocio config options from new release of ayon-ocio addon.

## Additional information
- Introduced new OCIO config for ACES 1.3 and 2.0 Studio.
- Updated paths with relevant labels and descriptions.
- Ensured compatibility with OCIO v2 requirements.

With this PR ayon-ocio >= 1.2.0 is required.

jakubjezek001 committed:
Add ACES 1.3 Studio OCIO config option

- Introduced new OCIO config for ACES 1.3 Studio.
- Updated paths with relevant labels and descriptions.
- Ensured compatibility with OCIO v2 requirements.

jakubjezek001 committed:
Update OCIO config path for clarity

Refined the OCIO built-in paths to improve readability. Added a comment to ignore line length warning for better code style adherence.

iLLiCiTiT commented:
Shouldn't happen some change in https://github.com/ynput/ayon-ocio addon and bump it's version and add it's version requirement to ayon-core package.py?

This seems to me like bump minor requirement...

jakubjezek001 commented:
> Because [ynput/OpenColorIO-Configs](https://github.com/ynput/OpenColorIO-Configs?rgh-link-date=2025-01-31T10%3A50%3A43Z) is not used anywhere in the process.

In that case I am not actually sure what is the procedure here. I believe we should not be adding anything to ayon-ocio, but I might we wrong. @philippe-ynput perhaps you might be interested about this.

jakubjezek001 committed:
Update OCIO config paths and compatible addons

- Added a new compatible addon with version requirement.
- Updated existing OCIO config paths for ACES 1.3 and added support for ACES 2.0.
- Adjusted labels and descriptions for clarity on OCIO versions.

jakubjezek001 committed:
Update OCIO config paths and add comments

- Added line length comments to OCIO config paths
- No functional changes, just improved code clarity

jakubjezek001 commented:
I've made the following improvements to the workflow:

- The required configuration files are now downloaded on demand through the ayon-ocio addon during the `create_package.py` process
- Modified the `BUILTIN_OCIO_ROOT` value to accommodate a more flexible workflow structure
  - The original value targeted a folder within the distributed zip file
  - The updated value removes the folder parent, allowing us to specify the parent directory for all downloaded source files

These changes enable greater flexibility in managing configuration resources while maintaining compatibility with existing systems.

jakubjezek001 committed:
Update OCIO config paths for cleaner structure

- Removed redundant directory from ACES 1.2 path
- Simplified Nuke default config path

jakubjezek001 committed:
Merge branch 'develop' into enhancement/adding-aces-1.3-ocio-support

iLLiCiTiT commented on server/settings/main.py:
```diff
@@ -72,12 +72,30 @@ def _fallback_ocio_config_profile_types():
+                "Aces 2.0 Studio OCIO config file. Requires OCIO v2.4.")
+        },
+        {
+            "value": "{BUILTIN_OCIO_ROOT}/aces_1.2/config.ocio",  # noqa: E501
```
```suggestion
            "value": "{BUILTIN_OCIO_ROOT}/aces_1.2/config.ocio",
```

jakubjezek001 committed:
Merge branch 'develop' into enhancement/adding-aces-1.3-ocio-support

jakubjezek001 committed:
Remove unnecessary E501 comments

jakubjezek001 committed:
Merge branch 'develop' into enhancement/adding-aces-1.3-ocio-support

iLLiCiTiT committed:
added aces subfolders

iLLiCiTiT commented on server/settings/main.py:
```diff
@@ -71,6 +71,24 @@ def _fallback_ocio_config_profile_types():
 
 def _ocio_built_in_paths():
     return [
+        {
```
Is the order ok? ACES 1.3, ACES 2.0, ACES 1.2, Nuke

jakubjezek001 replied:
I would say that it should be: ACES 2.0, ACES 1.3, ACES 1.2, Nuke

jakubjezek001 committed:
Merge branch 'develop' into enhancement/adding-aces-1.3-ocio-support

jakubjezek001 committed:
Updates OCIO config paths

Updates the built-in OCIO config paths to correct versioning and descriptions, ensuring accurate configuration options.

jakubjezek001 committed:
Merge branch 'develop' into enhancement/adding-aces-1.3-ocio-support

iLLiCiTiT commented on package.py:
```diff
@@ -10,6 +10,7 @@
 ayon_launcher_version = ">=1.0.2"
 ayon_required_addons = {}
 ayon_compatible_addons = {
+    "ayon_ocio": ">=1.2.0",
```
```suggestion
    "ayon_ocio": ">=1.2.1",
```
